### DISCUSSION TITLE
Capacity Information

### DISCUSSION THREAD

**Discussion by JonA02** [kurisubrooks/tfnsw-pids#5] [Ideas]
Hey there!

I know it's currently not in a working state (I.e. the CORS proxy) but do you have any plans for adding capacity information to this?

Cheers!

**Comment by kurisubrooks** [comment]
Capacity information is available in the API that we use, however the owner of the API said that TfNSW does not wish for it to be utilised in "unauthorised" third-party apps, so it is not supported at this time.

**Reply by JonA02** [reply]
Ahhhh that's fair!! Thanks anyways!! Cheers!
